### PR TITLE
docs: add 9.30 weekly report

### PR DESCRIPTION
**Pull request by lijing-21**
add 9.30 weekly report

### CODE REVIEW

File: website/blog/2021-09-30-weekly-report.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+![问题描述截图](https://static.apiseven.com/202108/1632799650125-14edb988-f2ad-434d-8d13-04ff3016eb5a.png)
+
+### Issue [#5108](https://github.com/apache/apisix/issues/5108)
+
+问题描述：如下，当在路由上启用请求验证插件:
+
+```
+curl http://127.0.0.1:9080/apisix/admin/routes/35 -H 'X-API-KEY: xxxxxxxxxxxxxxxxxxx' -X PUT -d '
+{
+"uri": "/products/create",
```
Comment (juzhiyuan) [inline website/blog/2021-09-30-weekly-report.md]: Please format this JSON data

File: website/blog/2021-09-30-weekly-report.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---
+title: "社区周报｜新晋 committer 一位，功能亮点更新进行中"
+keywords:
+- APISIX
+- 社区周报
+- 贡献者
+- Good first issue
```
Comment (yzeng25) [inline website/blog/2021-09-30-weekly-report.md]:
```
keywords:
- Apache APISIX
- APISIX
- API Gateway
- 社区周报
- 贡献者
```

File: website/blog/2021-09-30-weekly-report.md (hunk 1 above)
Comment (yzeng25) [reply website/blog/2021-09-30-weekly-report.md]: try json formatter online: https://jsonformatter.curiousconcept.com/#

```
{
    "uri":"/products/create",
    "plugins":{
        "request-validation":{
            "body_schema":{
                "type":"object",
                "required":[
                    "productName",
                    "price"
                ],
                "properties":{
                    "productName":{
                        "type":"string"
                    },
                    "price":{
                        "type":"number"
                    }
                }
            }
        }
    },
    "upstream":{
        "service_name":"PRODUCTSSERVICE",
        "type":"roundrobin",
        "discovery_type":"eureka"
    }
}’
```

File: website/blog/2021-09-30-weekly-report.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+问题描述：之前上游服务使用 IP 认证，实际的客户端 IP 是从 x-forwarded-for 请求头中获取的。现在需要改成网关 HMAC 认证，所以需要通过网关屏蔽上游 IP 认证。尝试通过代理重写插件修改 x-forwarded-for，但是没有生效：
+
+![问题描述截图](https://static.apiseven.com/202108/1632799650125-14edb988-f2ad-434d-8d13-04ff3016eb5a.png)
+
+### Issue [#5108](https://github.com/apache/apisix/issues/5108)
+
+问题描述：如下，当在路由上启用请求验证插件:
+
+```
```
Comment (yzeng25) [inline website/blog/2021-09-30-weekly-report.md]: "```shell"

File: website/blog/2021-09-30-weekly-report.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+"service_name": "PRODUCTSSERVICE",
+"type": "roundrobin",
+"discovery_type": "eureka"
+}
+}'
+```
+
+使用以下命令对其进行测试时，
+
+```
```
Comment (yzeng25) [inline website/blog/2021-09-30-weekly-report.md]: "```shell"

File: website/blog/2021-09-30-weekly-report.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+curl http://127.0.0.1:9080/products/create -X POST -d '{"product-Name":"Laptop","pri-ce":12345.00}'
+```
+
+得到以下默认信息：
+
+```
```
Comment (yzeng25) [inline website/blog/2021-09-30-weekly-report.md]: "```shell"

File: website/blog/2021-09-30-weekly-report.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+- [支持 10 亿日流量的基础设施：当 Apache APISIX 遇上腾讯](https://mp.weixin.qq.com/s?__biz=MzI1MDU3NjQ5OA==&mid=2247488353&idx=1&sn=4dc9f8d16b323f8b672cca9b565e0001&chksm=e981787cdef6f16a64b8af8d9b76d351a7838254f34a3d75514d9df9392c8e272087297b278e&token=434953675&lang=zh_CN#rd)：
+
+  通过阅读本文，您不仅可以了解网关是什么、网关模式对传统服务架构的改进，还可以了解腾讯 OTeam 诞生的原因，以及 Apache APISIX 是如何在腾讯内部落地的。
+
+- [使用 Apache APISIX 进行集中式身份认证及进阶玩法](https://mp.weixin.qq.com/s?__biz=MzI1MDU3NjQ5OA==&mid=2247488648&idx=1&sn=e197e8b03dad3c3c4d0e6712c4400a24&chksm=e9817f95def6f683f01152bd92275276fb980a0974598a66e0a53f7f0d26a22d1d3332299131&token=434953675&lang=zh_CN#rd)
+
+  本文介绍了 Apache APISIX 的身份认证功能，从重要性和玩法使用上进行了详细介绍和细节使用。
+
+- [基于 Apache APISIX，爱奇艺 API 网关的更新与落地实践](https://mp.weixin.qq.com/s?__biz=MzI1MDU3NjQ5OA==&mid=2247488681&idx=1&sn=bec77fe4042eebf03d1d0118b800e24a&chksm=e9817fb4def6f6a28e29807fd818513230286ac7c40df1a687e16c08e2dd9f61588536d42860&token=434953675&lang=zh_CN#rd)
```
Comment (yzeng25) [inline website/blog/2021-09-30-weekly-report.md]: try to replace these links to Apache APISIX Blog's links.

File: website/blog/2021-09-30-weekly-report.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+### Issue [#5080](https://github.com/apache/apisix/issues/5080)
+
+问题描述：之前上游服务使用 IP 认证，实际的客户端 IP 是从 x-forwarded-for 请求头中获取的。现在需要改成网关 HMAC 认证，所以需要通过网关屏蔽上游 IP 认证。尝试通过代理重写插件修改 x-forwarded-for，但是没有生效：
+
+![问题描述截图](https://static.apiseven.com/202108/1632799650125-14edb988-f2ad-434d-8d13-04ff3016eb5a.png)
+
+### Issue [#5108](https://github.com/apache/apisix/issues/5108)
+
+问题描述：如下，当在路由上启用请求验证插件:
```
Comment (yzeng25) [inline website/blog/2021-09-30-weekly-report.md]: "如下，当在路由上启用请求验证插件:" delete ":"

File: website/blog/2021-09-30-weekly-report.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+> 从 9.13 到 9.30, 有 32 位开发者为 Apache APISIX 提交了 93 个 commits。感谢这些小伙伴为 Apache APISIX 添砖加瓦，是你们的无私付出，让 Apache APISIX 项目变得更好！
+<!--truncate-->
+
+## 导语
+
+Apache APISIX 从开源第一天就以社区方式成长，迅速成为全世界最活跃的开源 API 网关项目。这些成就，离不开社区小伙伴们的共同奋斗。
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的每周进展，方便大家参与到 Apache APISIX 社区中来。
+
+我们还整理了一些适合新来社区的小伙伴们参加的 issue ！感兴趣的同学们，走过路过不要错过！
```
Comment (yzeng25) [inline website/blog/2021-09-30-weekly-report.md]: "issue ！" to "issue！", delete extra space

File: website/blog/2021-09-30-weekly-report.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+
+## 贡献者统计
+
+![本周贡献者名单](https://static.apiseven.com/202108/1632907894918-c455f40e-a175-4944-8fac-11c590d43786.jpg)
+
+![本周新晋贡献者](https://static.apiseven.com/202108/1632908362102-b0b665e2-f37f-4a82-b8a3-68914925b565.jpg)
+
+## Good first issue
+
+### Issue [#5080](https://github.com/apache/apisix/issues/5080)
```
Comment (yzeng25) [inline website/blog/2021-09-30-weekly-report.md]: remove the link in section's title
add link below the title

File: website/blog/2021-09-30-weekly-report.md [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+
+问题描述：之前上游服务使用 IP 认证，实际的客户端 IP 是从 x-forwarded-for 请求头中获取的。现在需要改成网关 HMAC 认证，所以需要通过网关屏蔽上游 IP 认证。尝试通过代理重写插件修改 x-forwarded-for，但是没有生效：
+
+![问题描述截图](https://static.apiseven.com/202108/1632799650125-14edb988-f2ad-434d-8d13-04ff3016eb5a.png)
+
+### Issue [#5108](https://github.com/apache/apisix/issues/5108)
```
Comment (yzeng25) [inline website/blog/2021-09-30-weekly-report.md]: remove the link in section's title
add link below the title

**Review by guoqqqi — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

File: website/blog/2021-09-30-weekly-report.md [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+                }
+            }
+        }
+    },
+    "upstream":{
+        "service_name":"PRODUCTSSERVICE",
+        "type":"roundrobin",
+        "discovery_type":"eureka"
+    }
+}’
```
Comment (liuxiran) [inline website/blog/2021-09-30-weekly-report.md]:
```suggestion
}'
```

**Review by yzeng25 — APPROVED** [review]

**Review by liuxiran — APPROVED** [review]

**Review by liuxiran — APPROVED** [review]

**Pull request merged**